ajax/libs/mousetrap/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mousetrap",
   "filename": "mousetrap.min.js",
-  "version": "1.4.6",
+  "version": "1.6.0",
   "description": "Simple library for handling keyboard shortcuts",
   "homepage": "http://craig.is/killing/mice",
```

Comment:
@x09326 Same as #9895. It will redirect to `https://craig.is/killing/mice`. I think we don't need to modify it.